docs/Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: pull docs docs-quick docs-no-pull docs-test docs-local-static
 
-IMAGE = grafana/grafana-docs-dev:latest
+IMAGE = grafana/grafana-docs-dev:latest # latest here refers to Docker, not the latest stable release of Grafana
```

Comment:
`latest` is typically the most cutting edge version of _something_. Technically it's the docker image so perhaps something along the lines of:
```
# Please note this refers to the docker image used for the documentation site 
# development environment. It does not refer to the latest version of Grafana.
# Please see https://hub.docker.com/r/grafana/grafana-docs-dev/tags for additonal tags
```
I wonder if the comment would be better placed above the line as well to follow a more standard commenting style?

Reply:
Please use the suggestion feature, so I can click the button.